combinations/hash.tsv

```diff
@@ -13,6 +13,8 @@ bwa-mem2=2.2.1,samtools=1.12	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.14	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.15	bgruening/busybox-bash:0.1	0
 bwa-mem2=2.2.1,samtools=1.15.1	bgruening/busybox-bash:0.1	0
+hisat2=2.2.0,samtools=1.10	bgruening/busybox-bash:0.1	0
```

Comment:
Are really both versions needed?

Reply:
Apparently is necessary for old containers, maybe @drpatelh can explain better.

Reply:
The changes on this PR should only add a single container to overcome [this](https://github.com/nf-core/rnaseq/pull/870#issue-1383647872) issue.    This is why I suggested the previous changes.